<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdNdeCoWidget</class>
 <widget class="QWidget" name="C_SdNdeCoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_SdNdeCoWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeSpiStandard" name="pc_Splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="pc_WiDevicesTree" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="stw_opensyde_gui_elements::C_OgePubGenericTextWithBackground" name="pc_PubOverview">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>-Overview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="stw_opensyde_gui::C_SdNdeCoConfigTreeView" name="pc_CoConfigTree"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pc_WiContent" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="stw_opensyde_gui::C_SdNdeCoOverviewWidget" name="pc_OverviewWidget" native="true"/>
       </item>
       <item>
        <widget class="stw_opensyde_gui::C_SdNdeCoManagerIntfWidget" name="pc_IntfSettingsWidget" native="true"/>
       </item>
       <item>
        <widget class="stw_opensyde_gui::C_SdNdeCoDeviceWidget" name="pc_DeviceWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeSpiStandard</class>
   <extends>QSplitter</extends>
   <header>C_OgeSpiStandard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeCoOverviewWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeCoOverviewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeCoManagerIntfWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeCoManagerIntfWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeCoDeviceWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeCoDeviceWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeCoConfigTreeView</class>
   <extends>QTreeView</extends>
   <header>C_SdNdeCoConfigTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubGenericTextWithBackground</class>
   <extends>QPushButton</extends>
   <header>C_OgePubGenericTextWithBackground.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
